subsys/net/l2/ethernet/ethernet.c

```diff
@@ -171,6 +171,7 @@ static inline bool eth_is_vlan_tag_stripped(struct net_if *iface)
 	return (net_eth_get_hw_capabilities(iface) & ETHERNET_HW_VLAN_TAG_STRIP);
 }
 
+#if defined(CONFIG_NET_IPV4) || defined(CONFIG_NET_IPV6)
```

Comment:
There's a symbol for this combination

```suggestion
#if defined(CONFIG_NET_IP)
```

Reply:
Ugh, it looks like that doesn't compile.

How about I keep `#if defined(CONFIG_NET_IPV4) || defined(CONFIG_NET_IPV6)` to be consistent with the calling location: https://github.com/zephyrproject-rtos/zephyr/blob/9757ffa5fa46612737bc4814d373443ba0d043a1/subsys/net/l2/ethernet/ethernet.c#L422?

I can create a followup PR to replace with `CONFIG_NET_IP`.